src/__tests__/fixtures/Comp.svelte

```diff
@@ -1,23 +1,17 @@
 <svelte:options accessors />
 
 <script>
-  import { getContext } from 'svelte'
```

Comment:
Duplicate context testing between `render.test.js` and `context.test.js`. Since `context.test.js` has its own `Context.svelte` fixture, I removed context stuff from here